### ISSUE TITLE
It seems I've been able to kick myself out of my own project

### ISSUE THREAD

**Issue by jnavila** [WeblateOrg/weblate#6187]
**Describe the issue**

On https://hosted.weblate.org/projects/git-manpages, I can no longer access the administration screens of the project even though I'm the owner.

I was trying to kick out a user who keeps on adding/deleting translations (while on the history page) and I'm not sure but I may have selected my own changes. This is quite dangerous that the changes of owners can still present the ban button. 

**I already tried**

Describe the steps you tried to solve the problem yourself.

- [x] I've navigated everywhere in the project without any hint at how to reinstate me as an administrator, but did not find any way. This is a good thing indeed.
- [x] I've logged out/in in case this was attached to the session.

**To Reproduce the issue**

1. As an administrator, Go to 'history page'
2. Select a change, you've made
3. Click on 'ban' button
4. You're kicked out of your own project

**Expected behavior**

The ban button should not be displayed for the administrator's own changes. That's part of preventing administrators to shoot in their own foot.

**Server configuration and status**

hosted.weblate.org as of 2021-06-16

**Weblate deploy checks**

**Additional context**

I have (obviously) no low level credence to make changes in the DB. Could somebody from weblate.org be kind enough to fix this? I promise, I won't do it again :wink:

**Comment by nijel** [comment]
I've just unblocked you :-).

**Comment by jnavila** [comment]
Thank you 🤡
